docs/mkdocs/docs/development.md

```
@@ -0,0 +1,380 @@
+cd .devcontainer
+sudo apt install --upgrade $(cat apt.in)
+sudo ./apt-compile.sh
+```
+
+## Limitations
+
+### glibc 2.35+
+
+Wheels in [`cadquery-ocp`] v7.7.2 require glibc to be v2.35 or newer:
+
+```
```

Markdownlint (0.37.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)